### DISCUSSION TITLE
different OS

### DISCUSSION THREAD

**Discussion by rayr007** [abiosoft/colima#214] [Q&A]
Is there a way to start colima with ubuntu instead of alpine?

**Comment by abiosoft** [comment]
Colima at the moment uses a custom lightweight Alpine image with bundled dependencies specifically for running docker/containerd/kubernetes. Therefore direct user interactions with the VM is expected to be minimal.

Which of the following suits you better?

1. I want Colima as it is with container runtimes, I simply want to use Ubuntu (or another distro) instead of Alpine.
2. I do not care about container runtimes, I want an experience like WSL (on Windows) and would install anything I need manually.

**Reply by rayr007** [reply]
It was 1 that I am looking for. Was having issues getting my container setup moved over from docker-desktop. Specifically, one built on oraclelinux with an Oracle 19.3 DB installed in it. This worked fine on windows in WSL with Ubuntu so I thought perhaps Oracle was being extra fussy. However, I tried again after giving colima more RAM and got it to work.

**Reply by StasSuslov** [reply]
Guys, hello! Is there any way to specify an OS? For example, I want to try Alpine instead of Ubuntu.
